pychunkedgraph/backend/history.py

```diff
@@ -351,4 +351,26 @@ def get_all_log_entries(cg_instance):
             )
         except KeyError:
             continue
-    return log_entries
\ No newline at end of file
+    return log_entries
+
+def get_tabular_changelog_recent(cg_instance, start_time_int):
```

Comment:
this bypasses all the logic that is already in place. it appears to be a `_build_tabular_changelog` lite and should be incorporated into `SegmentHistory`

Reply:
Yes, this is mostly just _build_tabular_changelog. I copied it out as its own function because it's global, and doesn't depend on a specific root ID (and therefore would not have an instance of SegmentHistory to work with). Do you know of a better way to structure this?

Reply:
Yeah, I see the problem. This is quite minimal and the bulk of the logic is handled by the chunkedgraph already. I would move this into `common.py` for now. We need a proper abstraction for general history but this is not the PR to do that.